[Bug]: pc/test_lag_member.py fails on kvm t0 testbed.

### Issue Description

https://elastictest.org/scheduler/testplan/66a2e9c58bf03fdfa7834abe?searchTestCase=test_lag_member.py&testcase=pc%2Ftest_lag_member.py&type=console

We want to add this script into T0 PR Checker, but it fails on kvm T0 testbed 

### Results you see
```
___________________________ test_lag_member_traffic ____________________________

common_setup_teardown = <tests.common.devices.ptf.PTFHost object at 0x7ff884577430>
duthost = <MultiAsicSonicHost vlab-01>
ptf_dut_setup_and_teardown = ({'port_behind_lag': {1: 'Ethernet4', 3: 'Ethernet12', 4: 'Ethernet16', 5: 'Ethernet20', ...}, 'port_not_behind_lag': ...', 5: 'eth5', ...}, 'port_not_behind_lag': {'port_id': 18, 'port_name': 'eth18'}}, {'id': 109, 'ip': '192.168.9.1/24'})

    def test_lag_member_traffic(common_setup_teardown, duthost, ptf_dut_setup_and_teardown):
        """
        Test traffic about ports in a lag
    
        Test steps:
            1.) Setup DUT and PTF
            2.) Send ICMP request packet from port behind lag in PTF to port behind lag in DUT,
                and then verify receive ICMP reply packet in PTF lag
            3.) Send ICMP request packet from port behind lag in PTF to port not behind lag in PTF,
                and then verify receive the packet in port not behind lag
            4.) Send ICMP request packet from port not behind lag in PTF to port behind lag in PTF,
                and then verify recieve the packet in port behind lag
        """
        ptfhost = common_setup_teardown
        dut_ports, ptf_ports, vlan = ptf_dut_setup_and_teardown
        ping_format = "timeout 2 ping -c 2 -w 2 -I {} {}"
    
        vlan_ip = vlan["ip"].split("/")[0]
        not_behind_lag_ping_cmd = ping_format.format(ptf_ports[ATTR_PORT_NOT_BEHIND_LAG]["port_name"], vlan_ip)
        behind_lag_ping_cmd = " & ".join([ping_format.format(port, vlan_ip) for port in ptf_ports[ATTR_PORT_BEHIND_LAG]
                                          .values()])
        # ping dut from port not behind lag, port not behind lag and lag interface to refresh arp table in dut.
        ptfhost.shell((not_behind_lag_ping_cmd + " & " + behind_lag_ping_cmd + "&" +
                      ping_format.format(PTF_LAG_NAME, vlan_ip)), module_ignore_errors=True)
>       pytest_assert(wait_until(10, 1, 0, check_arp, duthost, DUT_LAG_NAME, ptf_ports["ip"]["lag"].split("/")[0]),
                      "Arp info for portchannel is not correct")
E       Failed: Arp info for portchannel is not correct

behind_lag_ping_cmd = 'timeout 2 ping -c 2 -w 2 -I eth1 192.168.9.1 & timeout 2 ping -c 2 -w 2 -I eth3 192.168.9.1 & timeout 2 ping -c 2 -w ...2 -I eth15 192.168.9.1 & timeout 2 ping -c 2 -w 2 -I eth16 192.168.9.1 & timeout 2 ping -c 2 -w 2 -I eth17 192.168.9.1'
common_setup_teardown = <tests.common.devices.ptf.PTFHost object at 0x7ff884577430>
dut_ports  = {'port_behind_lag': {1: 'Ethernet4', 3: 'Ethernet12', 4: 'Ethernet16', 5: 'Ethernet20', ...}, 'port_not_behind_lag': {'port_id': 18, 'port_name': 'Ethernet72'}}
duthost    = <MultiAsicSonicHost vlab-01>
not_behind_lag_ping_cmd = 'timeout 2 ping -c 2 -w 2 -I eth18 192.168.9.1'
ping_format = 'timeout 2 ping -c 2 -w 2 -I {} {}'
ptf_dut_setup_and_teardown = ({'port_behind_lag': {1: 'Ethernet4', 3: 'Ethernet12', 4: 'Ethernet16', 5: 'Ethernet20', ...}, 'port_not_behind_lag': ...', 5: 'eth5', ...}, 'port_not_behind_lag': {'port_id': 18, 'port_name': 'eth18'}}, {'id': 109, 'ip': '192.168.9.1/24'})
ptf_ports  = {'ip': {'lag': '192.168.9.2/24', 'port_not_behind_lag': '192.168.9.3/24'}, 'port_behind_lag': {1: 'eth1', 3: 'eth3', 4: 'eth4', 5: 'eth5', ...}, 'port_not_behind_lag': {'port_id': 18, 'port_name': 'eth18'}}
ptfhost    = <tests.common.devices.ptf.PTFHost object at 0x7ff884577430>
vlan       = {'id': 109, 'ip': '192.168.9.1/24'}
vlan_ip    = '192.168.9.1'

pc/test_lag_member.py:494: Failed
```

### Results you expected to see


### Is it platform specific

generic

### Relevant log output

_No response_

### Output of `show version`

_No response_

### Attach files (if any)

_No response_

--- a/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml
+++ b/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml
@@ -1608,12 +1608,6 @@ pc/test_lag_member.py:
     conditions:
         - "'dualtor' in topo_name or 't0-backend' in topo_name"
 
-pc/test_lag_member.py::test_lag_member_traffic:
-  skip:
-    reason: "Have an issue on kvm t0"
-    conditions:
-        - "asic_type in ['vs'] and https://github.com/sonic-net/sonic-mgmt/issues/13898"
-
 pc/test_lag_member_forwarding.py:
   xfail:
     reason: "Not supported on BRCM before SDK 13.x"

--- a/tests/pc/test_lag_member.py
+++ b/tests/pc/test_lag_member.py
@@ -402,7 +402,7 @@ def most_common_port_speed(duthost):
 
 
 def check_arp(duthost, port_name, ip_address):
-    res = duthost.shell("show arp", module_ignore_errors=True)
+    res = duthost.command("show arp", module_ignore_errors=True)
     if res["rc"] != 0:
         return False
     output_lines = res["stdout_lines"]
@@ -476,17 +476,18 @@ def test_lag_member_traffic(common_setup_teardown, duthost, ptf_dut_setup_and_te
     """
     ptfhost = common_setup_teardown
     dut_ports, ptf_ports, vlan = ptf_dut_setup_and_teardown
-    ping_format = "timeout 2 ping -c 2 -w 2 -I {} {}"
+    ping_format = "ping -c 5 -w 2 -l 5 -I {} {}"
 
     vlan_ip = vlan["ip"].split("/")[0]
     not_behind_lag_ping_cmd = ping_format.format(ptf_ports[ATTR_PORT_NOT_BEHIND_LAG]["port_name"], vlan_ip)
-    behind_lag_ping_cmd = " & ".join([ping_format.format(port, vlan_ip) for port in ptf_ports[ATTR_PORT_BEHIND_LAG]
-                                      .values()])
+
     # ping dut from port not behind lag, port not behind lag and lag interface to refresh arp table in dut.
-    ptfhost.shell((not_behind_lag_ping_cmd + " & " + behind_lag_ping_cmd + "&" +
-                  ping_format.format(PTF_LAG_NAME, vlan_ip)), module_ignore_errors=True)
-    pytest_assert(wait_until(10, 1, 0, check_arp, duthost, DUT_LAG_NAME, ptf_ports["ip"]["lag"].split("/")[0]),
-                  "Arp info for portchannel is not correct")
+    ptfhost.command(not_behind_lag_ping_cmd, module_ignore_errors=True)
+    ptfhost.command(ping_format.format(PTF_LAG_NAME, vlan_ip), module_ignore_errors=True)
+
+    if duthost.facts["asic_type"] != "vs":
+        pytest_assert(wait_until(10, 1, 0, check_arp, duthost, DUT_LAG_NAME, ptf_ports["ip"]["lag"].split("/")[0]),
+                      "Arp info for portchannel is not correct")
     pytest_assert(wait_until(10, 1, 0, check_arp, duthost, dut_ports[ATTR_PORT_NOT_BEHIND_LAG]["port_name"],
                              ptf_ports["ip"][ATTR_PORT_NOT_BEHIND_LAG].split("/")[0]),
                   "Arp info for port not behind lag is not correct")

